geode-client-native/src/cppcache/impl/ClientMetadataService.cpp

```diff
@@ -1,925 +1 @@

        CacheableHashSetPtr keys;

        if (iter == serverToKeysMap->end()) {

          keys = CacheableHashSet::create();

        }

        else {

          keys = iter.second();

        }

        HashMapT<CacheableInt32Ptr, CacheableHashSetPtr>::Iterator bucketToKeysiter = bucketToKeysMap->find(*bucket);

        if (bucketToKeysiter !=  bucketToKeysMap->end()) {

          CacheableHashSetPtr bkeys = bucketToKeysiter.second();

          for (CacheableHashSet::Iterator itr = bkeys->begin(); itr != bkeys->end(); ++itr) {

            keys->insert(*itr);

          }

        }

        serverToKeysMap->insert(serverLocation, keys);

      }

    }

    delete bucketToKeysMap;

    delete serverToBuckets;

    return serverToKeysMap;

}



BucketServerLocationPtr ClientMetadataService::findNextServer

  (HashMapT<BucketServerLocationPtr, CacheableHashSetPtr>* serverToBucketsMap, CacheableHashSetPtr& currentBucketSet) {



    BucketServerLocationPtr serverLocation;

    int max = -1;

    std::vector<BucketServerLocationPtr> nodesOfEqualSize;

    for(HashMapT<BucketServerLocationPtr, CacheableHashSetPtr>::Iterator itr = serverToBucketsMap->begin(); itr != serverToBucketsMap->end(); ++itr) {

      CacheableHashSetPtr buckets = CacheableHashSet::create();

      CacheableHashSetPtr sBuckets = itr.second();



      for(CacheableHashSet::Iterator sItr = sBuckets->begin(); sItr != sBuckets->end(); ++sItr) {

        buckets->insert(*sItr);

      }



      LOGDEBUG("ClientMetadataService::findNextServer currentBucketSet->size() = %d  bucketSet->size() = %d ", currentBucketSet->size(), buckets->size());



      for(CacheableHashSet::Iterator currentBucketSetIter = currentBucketSet->begin(); currentBucketSetIter != currentBucketSet->end(); ++currentBucketSetIter) {        

        buckets->erase(*currentBucketSetIter);

        LOGDEBUG("ClientMetadataService::findNextServer bucketSet->size() = %d ",  buckets->size());        

      }



      int size = buckets->size();

      if (max < size) {

        max = size;

        serverLocation = itr.first();

        nodesOfEqualSize.clear();

        nodesOfEqualSize.push_back(serverLocation);

      }

      else if (max == size){

        nodesOfEqualSize.push_back(serverLocation);

      }

    }



    int nodeSize = nodesOfEqualSize.size();

    if(nodeSize > 0) {

      RandGen randgen;

      int random = randgen(nodeSize);

      return nodesOfEqualSize.at(random);

    }

    return NULLPTR;

}



bool ClientMetadataService::AreBucketSetsEqual(CacheableHashSetPtr& currentBucketSet, CacheableHashSetPtr& bucketSet) {



  int32_t currentBucketSetSize = currentBucketSet->size();

  int32_t bucketSetSetSize = bucketSet->size();



  LOGDEBUG("ClientMetadataService::AreBucketSetsEqual currentBucketSetSize = %d bucketSetSetSize = %d ", currentBucketSetSize, bucketSetSetSize);



  if (currentBucketSetSize != bucketSetS
```

Comment:
It appears you have added new whitespace to every line in this file. Please correct.